### ISSUE TITLE
[ADDON] - Flux

### ISSUE THREAD

**Issue by kcoleman731** [aws-ia/terraform-aws-eks-blueprints#432]
Add support for Flux as an add-on 

https://github.com/aws-ia/terraform-aws-eks-blueprints/projects/1#card-80854259

**Comment by atali** [comment]
is it possible to add the same behavour as argocd with the flag [argocd_manage_add_ons](https://github.com/aws-ia/terraform-aws-eks-blueprints/tree/main/modules/kubernetes-addons#input_argocd_manage_add_ons)

**Comment by kcoleman731** [comment]
@atali yep that is what we will look to do. I don't believe we will be able to pass values from from resource created via TF to Flux however like we do with the ArgoCD application values map. We refer to this functionality as the [Gitops bridge](https://aws-ia.github.io/terraform-aws-eks-blueprints/add-ons/#gitops-bridge)

**Comment by frank-bee** [comment]
Any news here regarding flux support ? :-)
